src/lib/stitching/gravity/stitching.ts

```diff
@@ -81,29 +117,79 @@ export const gravityStitchingEnvironment = (
             })
           },
         },
-        formattedEndAt: {
+        distanceToOpen: {
+          fragment: gql`
+            ... on ViewingRoom {
+              startAt
+            }
+		  `,
+          resolve: ({ startAt: _startAt }, { short = false }) => {
+            if (_startAt === null) {
```

Comment:
```suggestion
            if (!_startAt) {
```

Reply:
I usually prefer the explicit way with null and undef, but sure.